app/script/links/LinkPreviewHelpers.js

```diff
@@ -22,16 +22,18 @@
 window.z = window.z || {};
 window.z.links = z.links || {};
 
+const codeBlockRegex = /(`|```)[^]*?\1/gm;
```

Comment:
@ffflorian thanks for the fix! I haven't tested it yet, but just want to confirm with you to be sure: code blocks aren't necessarily wrapped in one or three backticks, it could be four, five and more backticks. This works in Wire:

`````
````
aaa
```
bbb
````
`````

Reply:
Ah, you are right. Will be fixed in https://github.com/wireapp/wire-webapp/pull/5201.
Thanks!